src/strands/experimental/bidirectional_streaming/models/novasonic.py

```diff
@@ -67,7 +65,7 @@
 
 NOVA_AUDIO_OUTPUT_CONFIG = {
     "mediaType": "audio/lpcm",
-    "sampleRateHertz": 24000,
+    "sampleRateHertz": 16000,
```

Comment:
why did we update this? testing this branch, Nova Sonic is talking too fast to me